testing/cffi1/test_parse_c_type.py

```diff
@@ -164,6 +164,8 @@ def test_simple():
             ("long double", lib._CFFI_PRIM_LONGDOUBLE),
             (" float  _Complex", lib._CFFI_PRIM_FLOATCOMPLEX),
             ("double _Complex ", lib._CFFI_PRIM_DOUBLECOMPLEX),
+            ("_cffi_float_complex_t", lib._CFFI_PRIM_FLOATCOMPLEX),
+            (" _cffi_double_complex_t", lib._CFFI_PRIM_DOUBLECOMPLEX),
```

Comment:
Is the leading space intentional?

Reply:
Yes, absolutely.  It's a test, and there are already random spaces in the lines before.